controllers/v1/formsController.js

```diff
@@ -22,6 +22,219 @@ module.exports = class Forms extends Abstract {
+				"version": 0,
+				"deleted": false,
+				"type": "cccc",
+				"subType": "ppp",
+				"data": {
+					"name": "xyz"
+				},
+				"organizationId": 1,
+				"updatedAt": "2024-05-09T12:57:53.636Z",
+				"createdAt": "2024-05-09T12:53:09.920Z",
+				"__v": 0
+			}
+		}
+	*/
+
+	/**
+	 * updates form
+	 * @method
+	 * @name update
+	 * @param {String} req.params._id - form id.
+	 * @param {Object} req.body - request data.
+	 * @param {Number} req.userDetails.organizationId -organization id.
+	 * @returns {JSON} - forms updation response.
+	 */
+
+	async update(req) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				const updatedForm = await formsHelper.update(
+					req.params._id,
+					req.body,
+					req.userDetails.organizationId
+				)
+				return resolve(updatedForm)
+			} catch (error) {
+				return resolve({
+					status: error.status || httpStatusCode.internal_server_error.status,
+					message: error.message || httpStatusCode.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
+	}
+  	/**
+	 * @api {get} /project/v1/forms/read/:_id
+	 * @apiVersion 1.0.0
+	 * @apiName read
+	 * @apiGroup Forms
+	 * @apiParamExample {json} Request-Body:
+	 * @apiHeader {String} X-authenticated-user-token Authenticity token
+	 * @apiSampleRequest /project/v1/forms/read/663cc73584f1a0eb4e97e3db
+	 * @apiUse successBody
+	 * @apiUse errorBody
+	 * @apiParamExample {json} Request-Body:
+		{
+			"type" : "cccc",
+			"subType" : "ppp"
+		}
+
+    * @apiParamExample {json} Response:
+		{
+			"message": "Form fetched successfully",
+			"status": 200,
+			"result": {
+				"version": 0,
+				"deleted": false,
+				"_id": "663cc73584f1a0eb4e97e3db",
+				"type": "cccc",
+				"subType": "ppp",
+				"data": {
+					"name": "xyz"
+				},
+				"organizationId": 1,
+				"updatedAt": "2024-05-09T12:57:53.636Z",
+				"createdAt": "2024-05-09T12:53:09.920Z",
+				"__v": 0
+			}
+		}
+	*/
+
+	/**
+	 * reads form
+	 * @method
+	 * @name read
+	 * @param {String} req.params._id - form id.
+	 * @param {Object} req.body - request data.
+	 * @param {Number} req.userDetails.organizationId -organization id.
+	 * @param {String} req.userDetails.userToken -userToken.
+	 * @returns {JSON} - form object.
+	 */
+
+	async read(req) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				if (!req.params._id && Object.keys(req.body).length === 0) {
+					const formData = await formsHelper.readAllFormsVersion()
+					return resolve(formData)
+				} else {
+					const formData = await formsHelper.read(
+						req.params._id,
+						req.body,
+						req.userDetails.organizationId,
+						req.userDetails.userToken
+					)
+					return resolve(formData)
+				}
+			} catch (error) {
+        console.log(error)
+				return resolve({
+					status: error.status || httpStatusCode.internal_server_error.status,
+					message: error.message || httpStatusCode.internal_server_error.message,
+					errorObject: error,
+				})
```

Comment:
@borkarsaish65 validator are missing for all the  apis mentioned in this controller. Please check and add